kaznet/apps/main/api.py

```diff
@@ -53,6 +55,9 @@ def create_submission(ona_instance: object):
             ona_status=data[settings.ONA_STATUS_FIELD])
         validated_data['comments'] = str(
             data.get(settings.ONA_COMMENTS_FIELD, ""))
+        # indicate that the instance object's review status
+        # has already been synced
+        ona_instance.json["synced_with_ona_data"] = True
```

Comment:
Why don't we update the instance ? Seems to me like this is never updated outside this context, unless I'm missing something ?

Reply:
This was an oversight. `.save` should have been called here.

Reply:
@FrankApiyo I'll open an issue to change it( Please pick it up if you're free to do so, if not i'll work on it). It also seems like adding a `.save` there causes an infinite loop of sorts since `.save` calls this function through the `post_save` single.... Might need to find a better way to implement this sync feature.